src/NodaTime.Demo/PeriodDemo.cs

```diff
@@ -177,5 +177,28 @@ public void Between_LocalDateTimesWithGivenUnits()
             Assert.AreEqual(23, period.Hours);
             Assert.AreEqual("P2Y264DT23H", period.ToString());
         }
+
+        [Test]
+        public void Normalize()
+        {
+            var original = new PeriodBuilder { Weeks = 2, Days = 5 }.Build();
+
+            var actual = Snippet.For(original.Normalize());
+
+            Assert.AreEqual(
+                new PeriodBuilder { Days = 19 }.Build(), 
+                actual);
+        }
+
+        [Test]
+        public void ToDuration()
+        {
+            Period oneDayAsPeriod = Period.FromDays(1);
```

Comment:
Again, let's remove some of this whitespace to create a nice simple snippet.

Reply:
Fixed.